crates/util/src/command.rs

```diff
@@ -0,0 +1,32 @@
+use std::ffi::OsStr;
+
+#[cfg(target_os = "windows")]
+const CREATE_NO_WINDOW: u32 = 0x0800_0000_u32;
+
+#[cfg(target_os = "windows")]
+pub fn new_std_command(program: impl AsRef<OsStr>) -> std::process::Command {
+    use std::os::windows::process::CommandExt;
+
+    let mut command = std::process::Command::new(program);
```

Comment:
Can we reformat these to use `#[allow(unused_mut)]`, and then move the compile flag into the function body?